src/tunacode/tools/hashline_edit.py

```
@@ -260,7 +259,7 @@ async def _run_hashline_edit(
     cache_mutation()
     diff_text = _make_diff(filepath, original_lines, new_lines)
     output = f"{description}\n\n{diff_text}"
-    return await maybe_prepend_lsp_diagnostics(output, Path(filepath))
+    return output
```

Ruff (0.15.10)

[warning] Unnecessary assignment to `output` before `return` statement

Remove unnecessary assignment

(RET504)